### DISCUSSION TITLE
TV5 Monde - Listing

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#91] [Show and tell]
https://ott.tv5monde.com/Content/HLS/Live/channel(info)/variant.m3u8 TV5 Monde Info
https://ott.tv5monde.com/Content/HLS/Live/channel(style1)/variant.m3u8 TV5 Monde Style HD (403)
https://ott.tv5monde.com/Content/HLS/Live/channel(orient)/variant.m3u8 TV5 Monde Maghreb-Orient (403)
https://ott.tv5monde.com/Content/HLS/Live/channel(europe)/variant.m3u8 TV5 Monde Europe (403)

**Comment by ARAB-IPTV** [comment]
> https://ott.tv5monde.com/Content/HLS/Live/channel(info)/variant.m3u8 TV5 Monde Info https://ott.tv5monde.com/Content/HLS/Live/channel(style1)/variant.m3u8 TV5 Monde Style HD (403) https://ott.tv5monde.com/Content/HLS/Live/channel(orient)/variant.m3u8 TV5 Monde Maghreb-Orient (403) https://ott.tv5monde.com/Content/HLS/Live/channel(europe)/variant.m3u8 TV5 Monde Europe (403)

all the channels work with me expect tv5 monde europe .

**Reply by xbmckodi123** [reply]
Could you add TV5 Monde Info to the /regions/nam.m3u wich is Canada/USA and France country, I will appreciate and keep all channels and updates in one single github list, thank you very much

**Comment by techservicespro** [comment]
I'm Yann, a new member from Gabon. Please, for personal use, i want to play with VLC these .MPD links or make them work from .m3u8 extension.

TV5 Afrique -- TF1 SF -- LCI

https://afrique.tv5monde.com/direct-tv-streaming
https://ott.tv5monde.com/Content/DASH/Live/channel(afrique)/master.mpd

https://www.tf1.fr/tf1-series-films/direct
https://hd1-das-live-ssl.tf1.fr/video/fMeREcUV55y3qs7gX_bioA/1660096903/out/v1/d025b4f9239b4ce1aa53d1a7692cba0a/index.mpd

https://www.tf1info.fr/direct/
https://lci-das-live-ssl.tf1.fr/video/0FBIzUjUN6kt-KeF7lJUyA/1660094085/out/v1/3949f9922e0046a0926e11f8687e1c1a/index.mpd

I will take all you give me.

thank a lot.

**Reply by BellezaEmporium** [reply]
You can use streamlink for these kinds of links, or Kodi with the streamlink plugin in to make sure it works.
